frontends/web/src/routes/device/bitbox02/components/password-entry/password-entry.tsx

```diff
@@ -33,7 +33,7 @@ function replayVideo(ref: HTMLVideoElement): void {
   }
 }
 
-export function PasswordEntry({ children }: PropsWithChildren<IPasswordEntryProps>) {
+export const PasswordEntry: FunctionComponent = ({ children }) => {
```

Comment:
`<PasswordEntry />` never uses children props remove `FunctionComponent` and `children`.